aggregator/src/bin/aggregator.rs

```diff
@@ -63,6 +63,27 @@ async fn main() -> Result<()> {
             },
         );
 
+        let garbage_collector_future = {
+            let datastore = Arc::clone(&datastore);
+            async move {
+                if let Some(gc_config) = ctx.config.garbage_collection {
+                    let gc = GarbageCollector::new(
+                        datastore,
+                        gc_config.report_limit,
+                        gc_config.aggregation_limit,
+                        gc_config.collection_limit,
+                    );
+                    let mut interval = interval(Duration::from_secs(gc_config.gc_frequency_s));
```

Comment:
Are we operating on the assumption that will continue to operate one aggregator replica?

If not, could consider introducing jitter to prevent GC from running at the same time across all replicas. Any kubectl rollout will have all replicas coming up at roughly the same time.

Optional, I can't think of any real problem at the moment with GC running at the same time.

Reply:
We will almost surely run more than one replica; I am hoping to use configuration to run GC on only one replica. I _think_ we can use a `StatefulSet` for this (since that gives access to an "index", which allows us to control behavior on a replica-by-replica basis -- see e.g. https://itnext.io/kubernetes-statefulset-initialization-with-unique-configs-per-pod-7e02c01ada65). If that's too complicated or otherwise infeasible, we can always create a "leader" deployment with one replica, and a "followers" deployment with the remaining replicas.

In the long run, we might want to rethink this strategy (what if the one GC replica goes down?), but I think this is a pretty good starting point.